testing/go/functions_test.go

```diff
@@ -907,6 +907,58 @@ func TestSystemInformationFunctions(t *testing.T) {
 				},
 			},
 		},
+		{
+			Name: "pg_get_serial_sequence",
+			SetUpScript: []string{
+				`create table t0 (id INTEGER NOT NULL PRIMARY KEY);`,
+				`create table t1 (id SERIAL PRIMARY KEY);`,
+				`create sequence t2_id_seq START 1 INCREMENT 3;`,
+				`create table t2 (id INTEGER NOT NULL DEFAULT nextval('t2_id_seq'));`,
+				// TODO: ALTER SEQUENCE OWNED BY is not supported yet. When the sequence is created
```

Comment:
This isn't something that we're going to be supporting any time soon, since we removed ownership altogether.